### ISSUE TITLE
IncorrectLinesCountBeforeControlStructure false positive with parent:: call in ternary condition

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1070]
```php
class Foo extends Bar
{
    public static function method() : object
    {
        return true
            ? parent::foo()
            : self::bar();
    }
}
```

gives `Expected 1 lines before "parent", found -1.`

> SlevomatCodingStandard.Classes.ParentCallSpacing.IncorrectLinesCountBeforeControlStructure

**Comment by kukulich** [comment]
@simPod is it the right code? There’s no parent keyword...

**Comment by simPod** [comment]
@kukulich sorry, changed it in the last second to validate the bug. Updated.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/c57de703cb3dc051ad6a2c0083258036100eb566
